Engines/Wine/Verbs/FAudio/script.js

```diff
@@ -0,0 +1,68 @@
+include(["engines", "wine", "engine", "object"]);
+include(["engines", "wine", "plugins", "override_dll"]);
+include(["utils", "functions", "net", "resource"]);
+include(["utils", "functions", "filesystem", "files"]);
+
+/**
+* Verb to install FAudio
+* see: https://github.com/Kron4ek/FAudio-Builds
+* @returns {Wine} Wine object
+*/
+Wine.prototype.faudio = function () {
+    var faudioVersion = "19.01";
```

Comment:
Can we make the version selectable? E.g. pass a version string?

Reply:
Yes, but currently there is only one version ^^.

Reply:
I think it makes sense to prepare it already nevertheless.